extensions/sql-database-projects/src/models/project.ts

```diff
@@ -1351,6 +1359,11 @@ export class Project implements ISqlProject {
 		// If folder doesn't exist, create it
 		await fs.mkdir(absoluteFolderPath, { recursive: true });
 
+		// don't need to add the folder to the sqlproj if this is an msbuild sdk style project because globbing will get the folders
```

Comment:
I never quite understood what the Folders are for, are they just used for grouping the files in the tree view?

Reply:
yeah I think they're just for the UI, even though SSDT shows the folders even if the Folder entries aren't in the sqlproj. I think one use for them is for determining if empty folders or folders with all the contents excluded should be shown, but I'll look into that more later and handle those scenarios in a different PR.